src/models/Settings.js

```diff
@@ -156,7 +156,10 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'consent.cancel': 'function',
 
       //IDP Discovery
-      'idpDiscovery.requestContext': 'string'
+      'idpDiscovery.requestContext': 'string',
+
+      //Colors
+      'colors.brand': 'string'
```

Comment:
Does our model validation fail if you pass in `rgb(255,0,0)`? If not, maybe we check to see if the color is prepended with a `#`.

Reply:
Will add model validation 👍